docs/mms/index.md

```diff
@@ -0,0 +1,91 @@
+
+```bash
+mxnet-model-server --help
+```
+
+Here is an easy example for serving an object classification model. You can use any URI and the model will be downloaded first, then served from that location:
+
+```bash
+mxnet-model-server \
+  --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+```
+
+
+### Test Inference on a Model
+
+Assuming you have run the previous `mxnet-model-server` command to start serving the object classification model, you can now upload an image to its `predict` REST API endpoint. The following will download a picture of a kitten, then upload it to the prediction endpoint.
+
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+curl -X POST http://127.0.0.1:8080/squeezenet/predict -F "data=@kitten.jpg"
+```
+
+For more examples of serving models visit the following resources:
+
+* [Quickstart: Model Serving](https://github.com/awslabs/mxnet-model-server/README.md#serve-a-model)
+* [Running the Model Server](https://github.com/awslabs/mxnet-model-server/docs/server.md)
+
+## Create a Model Archive
+
+Creating a model archive involves rounding up the required model artifacts, then using the `mxnet-model-export` command line interface. As the process for creating archives is likely to evolve as the project adds features it is recommended that you review the following resources to get the latest instructions:
+
+* [Quickstart: Export a Model](https://github.com/awslabs/mxnet-model-server/README.md#export-a-model)
+* [Model Artifacts](https://github.com/awslabs/mxnet-model-server/docs/export_model_file_tour.md)
+* [Loading and Serving Gluon Models](https://github.com/awslabs/mxnet-model-server/tree/master/examples/gluon_alexnet)
+* [Creating a MMS Model Archive from an ONNX Model](https://github.com/awslabs/mxnet-model-server/docs/export_from_onnx.md)
+
+
+## Using Containers
+
+Using Docker or other container services with MMS is a great way to scale your inference applications. It is recommended that you review the following resources for more information:
+
+* [Docker Quickstart](https://github.com/awslabs/mxnet-model-server/docker/README.md)
+* [MMS on Fargate](https://github.com/awslabs/mxnet-model-server/docs/mms_on_fargate.md)
+* [Optimized Configurations](https://github.com/awslabs/mxnet-model-server/docs/optimized_config.md)
+
+
+## Community & Contributions
+
+The MMS project is open to contributions from the community. If you like the idea of a simple serving solution for your models and would like to provide feedback, suggest features, or even jump in and contribute code or examples, please visit the [project page on GitHub](https://github.com/awslabs/mxnet-model-server). You can create an issue there, or join the discussion on the forum.
+
+* [MXNet Forum - MMS Discussions](https://discuss.mxnet.io/c/mxnet-model-server)
+
+
+## Further Reading
```

Comment:
Probably we should also reference AWS blogpost on using MMS with Fargate for productionization of DL model?

Reply:
added link to blog